portfolio/src/main/webapp/script.js

```diff
@@ -15,14 +15,15 @@
 /**
  * Adds a random greeting to the page.
  */
+document.getElementsByTagName("h1")[0].style.fontSize = "6vw";
 function addRandomGreeting() {
   const greetings =
       ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
 
   // Pick a random greeting.
   const greeting = greetings[Math.floor(Math.random() * greetings.length)];
-
+  console.log("greeting: " + greeting);
```

Comment:
nit: Left over from debugging?

Reply:
yep, was part of the intro exercise they had us do